data/src/test/java/org/apache/iceberg/io/TestTaskEqualityDeltaWriter.java

```diff
@@ -462,7 +468,13 @@ public void delete(Record row) throws IOException {
     }
 
     public void deleteKey(Record key) throws IOException {
-      deltaWriter.deleteKey(key);
+      // Project onto just the equality delete schema.
+      GenericRecord keyProjection = eqDeleteRecord.copy();
+      for (String field : eqDeleteColumns) {
+        keyProjection.setField(field, key.getField(field));
+        keyProjection.setField(field, key.getField(field));
```

Comment:
Why do we have two same sentence here ?

Reply:
Woops. I just got a new keyboard and I've been accidentally pressing a hot key that duplicates the current line of text. Since this doesn't change the semantics of the test, I missed it. Good catch!

Reply:
Updated.